src/main/java/Game.java

```diff
@@ -1,65 +1,185 @@
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.util.ArrayList;
+import java.util.List;
+
 public class Game {
-	
-	private Player[] gamePlayer;
+
 	private static Map map;
-	private static java.util.Date timestamp;
+	private static int PORT;
+	private boolean isBusy; //Spin lock in case 2 commands at same time.
+	private static String north = "north";
+	private static String south = "south";
+	private static String west = "west";
+	private static String east = "east";
+	private static List<PlayerThread> users;
 
-	Game(){
-		gamePlayer = new Player[4];
+	Game() {
 		map = new Map();
-		timestamp = new java.util.Date();
-	}
-	
-	void startGame() {
-		//For the purpose of this game, we will be using only 1 player. multiplayer is easily expandable.
-		// If not, for each player you would have to open a new thread and window for them to play. OR network it.
```

Comment:
This is understandable and helps explain the decisions you made while building your system concisely. This decision did help limit your system to what was necessary.

Reply:
This is an example of the key concept that as a developer you should steer clear from implementing things that you might need or want in the future but that dont really need now.